source/guide/3.2/plugin-hostpool.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: bt_wiki
+title: Host Pool Plugin
+category: Plugins
+publish: true
+abstract: Cloudify Host Pool plugin description and configuration
+pageord: 600
+---
+{%summary%}
+{%endsummary%}
+
+
+# Description
+
+The Host Pool plugin is an interface to [Host Pool service](https://github.com/cloudify-cosmo/cloudify-host-pool-service).
+
+Host Pool Service allows to use an existing environment infrastructure for deploying services and applications.
```

Comment:
also this is a bit vague. we should clarify that all the host pool service does is allow for the usage of existing machines. not an entire infrastructure